protobuf/proto/crux.proto

```diff
@@ -629,8 +629,8 @@ message ContainerStatusItem {
   string name = 101;
   string command = 103;
   google.protobuf.Timestamp createdAt = 104;
-  ContainerStatus status = 105; // The 'State' of the container (Created, Running, etc)
-  string state = 106; // The 'Status' of the container ("Created 1min ago", "Exited with code 123", etc)
+  ContainerState state = 105; // The 'State' of the container (Created, Running, etc)
```

Comment:
Add comment above the item, there is a proto based docs generator, that way it will be moved into the generated docs too surely, but if comment follows the field I am not so sure.